ledger/catchpointtracker_test.go

```diff
@@ -46,7 +47,7 @@ import (
 	"github.com/algorand/go-algorand/test/partitiontest"
 )
 
-func TestIsWritingCatchpointFile(t *testing.T) {
+func TestCatchpointIsWritingCatchpointFile(t *testing.T) {
```

Comment:
I renamed catchpointtracker_test.go tests to unify names and make it easier to run per-file